jbt-it/neomdb
# feat(deployment): Added deploy script

**How to test**:
- Execute the python script
- Follow the instructions in the command line
  - use the dev environment: This should built the client and server docker containers locally and start the application on localhost:9000
  - use test environment: This should prompt you for a version (use 0.3.4), then build the client and server images and push them to our docker regsitry automatically
  -> Log into docker hub (use credentials in keepass) and check if the correct image versions are uploaded

Closes #378
